test/integration/etcd/etcd_storage_path_test.go

```diff
@@ -95,20 +94,15 @@ func testEtcdStoragePathWithVersion(t *testing.T, v string) {
 		// Only test for beta and GA APIs with emulated version.
 		featuregatetesting.SetFeatureGateEmulationVersionDuringTest(t, feature.DefaultFeatureGate, version.MustParse(v))
 		featuregatetesting.SetFeatureGateDuringTest(t, feature.DefaultFeatureGate, "AllBeta", true)
-		// Feature Gates that are GA and depend directly on the API version to work can not be emulated in previous versions.
-		// Example feature:
-		// v1.x-2 : FeatureGate alpha , API v1alpha1/feature
-		// v1.x-1 : FeatureGate beta  , API v1beta1/feature
-		// v1.x   : FeatureGate GA    , API v1/feature
-		// The code in v1.x uses the clients with the v1 API, if we emulate v1.x-1 it will not work against apiserver that
-		// only understand v1beta1.
-		featuregatetesting.SetFeatureGateDuringTest(t, feature.DefaultFeatureGate, features.MultiCIDRServiceAllocator, false)
 	}
 
 	apiServer := StartRealAPIServerOrDie(t, func(opts *options.ServerRunOptions) {
 		// Disable alphas when emulating previous versions.
 		if v != componentbaseversion.DefaultKubeBinaryVersion {
+			opts.Options.GenericServerRunOptions.RuntimeConfigEmulationForwardCompatible = true
 			opts.Options.APIEnablement.RuntimeConfig["api/alpha"] = "false"
+			// these APIs are required to enable Beta features in the api server.
```

Comment:
this may require a TODO for removing once `MultiCIDRServiceAllocator` does not require to be emulated, that IIRC is 3 versions from now, 1.36

Reply:
I'll follow up on this

Reply:
ok, nevermind, this commit no longer exist, and is fixed in last commit